ngraph/core/src/pass/graph_rewrite.cpp

```diff
@@ -272,6 +272,19 @@ void pass::GraphRewrite::add_matcher(const shared_ptr<pattern::Matcher>& m,
     NGRAPH_SUPPRESS_DEPRECATED_END
 }
 
+void pass::GraphRewrite::set_pass_config(const std::shared_ptr<PassConfig>& rhs)
+{
+    auto pass_config = get_pass_config();
+    rhs->add_disabled_passes(*pass_config);
```

Comment:
The same comment as above.

Reply:
During offline discussion decided to keep it as is but add some explanation comments.